src/components/PathwayExercise/index.js

```diff
@@ -172,23 +172,23 @@ function PathwayExercise() {
   };
 
   useEffect(() => {
-    if(localStorage.getItem("studentAuth")|| (user && user?.data?.token)){
-      return
-    }else{
+    if (localStorage.getItem("studentAuth") || (user && user?.data?.token)) {
+      return;
+    } else {
       history.push("/login");
```

Comment:
Tumhen constant refer karnaa chahiye:
```suggestion
      history.push(PATHS.LOGIN);
```